### PR TITLE
updated shared modal window

### PR DESCRIPTION
**Pull request by MariyaSin**

### CODE REVIEW

**Review by BelehovEgor — APPROVED** [review]

File: src/GUI/Pages/Admin/Company/CreateDepartment.razor.cs [hunk 1]
```diff
@@ -18,19 +19,25 @@ private async Task HandleValidSubmit()
...
             {
                 _message = ex.Result.Message;
+                StateHasChanged();
             }
             catch (ApiException<OperationResultResponse> ex)
             {
                 _message = String.Join(" ", ex.Result.Errors);
+                StateHasChanged();
             }
             catch (ApiException ex)
```
Comment (djino97) [inline src/GUI/Pages/Admin/Company/CreateDepartment.razor.cs]: ex not used

File: src/GUI/Pages/Admin/User/CreateUser.razor.cs [hunk 2]
```diff
@@ -39,19 +40,24 @@ private async Task HandleValidSubmitAsync()
             try
             {
                 await userService.CreateUserAsync(_userData);
-                UriHelper.NavigateTo("/");
+                _message = "Successfully created";
+                _isSuccessOperation = true;
+                StateHasChanged();
```
Comment (djino97) [inline src/GUI/Pages/Admin/User/CreateUser.razor.cs]: Why be not moved StateHasShanged to method end?

**Review by djino97 — APPROVED** [review]

File: src/GUI/Pages/Admin/Company/CreateDepartment.razor.cs (hunk 1 above)
Comment (MariyaSin) [reply src/GUI/Pages/Admin/Company/CreateDepartment.razor.cs]: Use if something wrong

File: src/GUI/Pages/Admin/User/CreateUser.razor.cs (hunk 2 above)
Comment (MariyaSin) [reply src/GUI/Pages/Admin/User/CreateUser.razor.cs]: done

**Review by borovHokage — APPROVED** [review]

**Pull request merged**